cmd/liqonet/main.go

```diff
@@ -59,7 +67,7 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&runAs, "run-as", "tunnel-operator",
-		"The accepted values are: liqo-gateway, liqo-route, tunnelEndpointCreator-operator. The default value is \"tunnel-operator\"")
+		"The accepted values are: liqo-gateway, liqo-route, tunnelEndpointCreator-operator. The default value is \"liqo-gateway\"")
```

Comment:
I think this is an unwanted change.

Reply:
That was intentional, but I forgot to change the default value of `run-as` from `tunnel-operator` to `liqo-gateway`. Nice catch!